File: src/frame_a_to_frame_b_broadcaster.py
#!/usr/bin/env python
import roslib
import rospy
import tf
import time
import math

if __name__ == '__main__':
    rospy.init_node('frame_a_frame_b_broadcaster_node')

    time.sleep(2)

    broadcaster = tf.TransformBroadcaster()

    while (not rospy.is_shutdown()):
        translation = (1,2,3)
        rotation_quaternion = tf.transformations.quaternion_from_euler(0.2,0.3,0.1)
        current_time = rospy.Time.now()

        broadcaster.sendTransform(
            translation,
            rotation_quaternion,
            current_time,
            'frame_b', 'frame_a'
        )
        time.sleep(0.5)

    rospy.spin()

File: src/frame_a_to_frame_b_broadcaster.cpp
#include <ros/ros.h>
#include <tf/transform_broadcaster.h>

int main(int argc, char** argv){
    ros::init(argc, argv, "frame_a_frame_b_broadcaster");
    ros::NodeHandle node;

    tf::TransformBroadcaster br;
    tf::Transform transform;

    ros::Rate rate(2.0);
    while (node.ok()) {
        transform.setOrigin( tf::Vector(1,2,3) );
        transform.setRotation( tf::Quaternion(0, 0, 0, 1) );
        br.sendTransform(tf::stampedTransform(transform, ros::Time::now(),
            "frame_b","frame_a"));
        rate.sleep();
    }
    return 0;
};
